pagopa/io-app
# feat: [IOCOM-658] Remove the F24 section on cancelled SEND message

## Short description
This PR removes the F24 section on cancelled SEND messages

## List of changes proposed in this pull request
- used `isCancelled` to render the F24 section

## How to test
Using `io-dev-api-server` generate a cancelled SEND message with multiple F24. Check that the F24 section is not rendered
